docker-compose.yml

```diff
@@ -1,6 +1,20 @@
 version: '3.8'
 
 services:
+  app:
+    build:
+      context: .
+      dockerfile: ./.docker/app/Dockerfile
+    networks:
+      - mariadb
+    volumes:
+      - .:/app
```

Comment:
How does the bind volume behave here, since you added the files into the image in Dockerfile? Does it just mask the folder and files in the image with the ones from the host machine?

Reply:
This allows you to do make changes and see the changes right away. Copying to image is needed for installing dependencies.

Reply:
Yes, I was just curious as to how it behaves when it overwrites a folder in the image that already contains files.